com.getferdi.Ferdi.yaml

```diff
@@ -0,0 +1,48 @@
+app-id: com.getferdi.Ferdi
+base: org.electronjs.Electron2.BaseApp
+base-version: '19.08'
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: ferdi
+separate-locales: false
+rename-desktop-file: ferdi.desktop
+rename-icon: ferdi
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=xdg-download
+  - --filesystem=xdg-pictures:ro
+  - --filesystem=xdg-videos:ro
+  - --filesystem=xdg-documents:ro
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+modules:
+  - name: ferdi-desktop
+    buildsystem: simple
+    build-commands:
+      - ar x ferdi_*.deb
```

Comment:
Any reason for not building it instead of using the deb package?

Reply:
Generally no. I just went with what Franz did here. Building from source is absolutely an option.

Reply:
Alright I gave it a try but ran into way too many problems. Might try this some other time but I give up on this for now.